content/en/docs/languages/net/exporters.md

```diff
@@ -254,13 +275,14 @@ app.UseOpenTelemetryPrometheusScrapingEndpoint();
 await app.RunAsync();
 ```
 
-### Non-ASP.NET Core
+#### Non-ASP.NET Core {#prometheus-non-asp-net-core-usage}
 
 For applications not using ASP.NET Core, you can use the `HttpListener` version
-which is available in a different package:
+which is available in a
```

Comment:
could we include a warning that this component has no intention of getting PROD ready:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Prometheus.HttpListener#prometheus-exporter-httplistener-for-opentelemetry-net